content/en/docs/tasks/security/automtls/index.md

```diff
@@ -0,0 +1,289 @@
+$ kubectl create ns full
+$ kubectl apply -f <(istioctl kube-inject -f @samples/httpbin/httpbin.yaml@) -n full
+$ kubectl apply -f <(istioctl kube-inject -f @samples/sleep/sleep.yaml@) -n full
+{{< /text >}}
+
+`partial` namespace contains server workloads partially migrated to Istio. Only migrated one has
+sidecar injected, able to serve mutual TLS traffic.
+
+{{< text bash >}}
+$ kubectl create ns partial
+$ kubectl apply -f <(istioctl kube-inject -f @samples/httpbin/httpbin.yaml@) -n partial
+$ cat <<EOF | kubectl apply -n partial -f -
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: httpbin-nosidecar
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: httpbin
+        version: nosidecar
+    spec:
+      containers:
+      - image: docker.io/kennethreitz/httpbin
+        imagePullPolicy: IfNotPresent
+        name: httpbin
+        ports:
+        - containerPort: 80
+EOF
+{{< /text >}}
+
+`legacy` namespace contains the workloads and none of them have Envoy sidecar.
+
+{{< text bash >}}
+$ kubectl create ns legacy
+$ kubectl apply -f @samples/httpbin/httpbin.yaml@ -n legacy
+$ kubectl apply -f @samples/sleep/sleep.yaml@ -n legacy
+{{< /text >}}
+
+Last we deploy two `sleep` workloads, one has sidecar and one does not.
+
+{{< text bash >}}
+$ kubectl apply -f <(istioctl kube-inject -f @samples/sleep/sleep.yaml@) -n full
+$ kubectl apply -f @samples/sleep/sleep.yaml@ -n legacy
+{{< /text >}}
+
+You can confirm the deployments in all namespaces.
+
+{{< text bash >}}
+$ kubectl get pods -n full
+$ kubectl get pods -n partial
+$ kubectl get pods -n legacy
+NAME                      READY   STATUS    RESTARTS   AGE
+httpbin-dcd949489-5cndk   2/2     Running   0          39s
+sleep-58d6644d44-gb55j    2/2     Running   0          38s
+NAME                       READY   STATUS    RESTARTS   AGE
+httpbin-6f6fc94fb6-8d62h   1/1     Running   0          10s
+httpbin-dcd949489-5fsbs    2/2     Running   0          12s
+NAME                       READY   STATUS    RESTARTS   AGE
+httpbin-54f5bb4957-lzxlg   1/1     Running   0          6s
+sleep-74564b477b-vb6h4     1/1     Running   0          4s
+{{< /text >}}
+
+You should also verify that there is a default mesh authentication policy in the system, which you can do as follows:
+
+{{< text bash >}}
+$ kubectl get policies.authentication.istio.io --all-namespaces
+NAMESPACE      NAME                          AGE
+istio-system   grafana-ports-mtls-disabled   5m
+$ kubectl get meshpolicies.authentication.istio.io
+NAME      AGE
+default   3m
+{{< /text >}}
+
+Last but not least, verify that there are no destination rules that apply on the example services. You can do this by checking the `host:` value of
+ existing destination rules and make sure they do not match. For example:
+
+{{< text bash >}}
+$ kubectl get destinationrules.networking.istio.io --all-namespaces -o yaml | grep "host:"
+    host: kubernetes.default.svc.cluster.local
+    host: '*.local'
```

Comment:
This actually match every service. Are you sure this shows up in the default setup?

Reply:
Right. This is a regression in the new `istioctl manifest` compared with the old demo installer profile. I filed https://github.com/istio/istio/issues/18654 for tracking. But this is accurate at this moment.

*.local DR has a DISABLED TLS mode.